veggieseasons/lib/widgets/veggie_card.dart

```diff
@@ -7,6 +7,69 @@ import 'package:veggieseasons/data/veggie.dart';
 import 'package:veggieseasons/screens/details.dart';
 import 'package:veggieseasons/styles.dart';
 
+/// A Card-like Widget that responds to tap events by animating changes to its
+/// elevation and invoking an optional [onPressed] callback.
+class PressableCard extends StatefulWidget {
+  const PressableCard({
+    @required this.child,
+    this.borderRadius = const BorderRadius.all(Radius.circular(10.0)),
+    this.upElevation = 2.0,
+    this.downElevation = 0.0,
+    this.shadowColor = CupertinoColors.black,
+    this.duration = const Duration(milliseconds: 100),
+    this.onPressed,
+    Key key,
+  })  : assert(child != null),
+        super(key: key);
```

Comment:
you probably need to do more null asserts

Reply:
A year of dart and I'm still not used to nullable by default. 😄 

Fixed.